app/Resources/FOSUserBundle/views/layout.html.twig

```diff
@@ -0,0 +1,16 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}
+    {{ 'add translation here' }}
+{% endblock %}
+
+{% block body %}
+    <br>
```

Comment:
I added padding-top to container class instead.
If you have a better/alternate solution, i'm all ears.